src/components/Avatar.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
+  import { t } from 'svelte-i18n';
+  import LogoGitHub32 from 'carbon-icons-svelte/lib/LogoGithub32';
+  import LogoLinkedIn32 from 'carbon-icons-svelte/lib/LogoLinkedin32';
+  import LogoTwitter32 from 'carbon-icons-svelte/lib/LogoTwitter32';
+  import Earth32 from 'carbon-icons-svelte/lib/Earth32';
+
+  // eslint-disable-next-line
+  export let avatar;
+</script>
+
+<div class="w-full lg:w-2/4 py-12 flex justify-center px-6">
+  <div class="w-full h-full lg:h-64 px-2 rounded overflow-hidden shadow-lg flex flex-col lg:flex-row items-center">
+    <div class="flex justify-center w-full">
+      <img class="w-24 h-24 rounded-full" src="{avatar.img}" alt="Avatar of {avatar.name}" />
+    </div>
+    <div class="flex flex-col">
+      <div class="px-6 py-4">
+        <div class="font-bold text-xl mb-2">{avatar.name}</div>
+        <p class="text-gray-700 text-base">{$t(avatar.bio)}</p>
+      </div>
+    </div>
+    <div class="px-6 pt-4 pb-2 flex flex-row space-x-4 lg:flex-col lg:space-y-4 lg:space-x-reverse">
+      <a href="{avatar.links.portfolio}">
```

Comment:
Why not a each loop with a component instantiation like I did here: https://github.com/Los-Crackitos/loscrackitos.codes/pull/4/files#diff-b4e77dfd35323001ce39e7ee445905daR22